plugins/home/src/homePageComponents/WelcomeTitle/WelcomeTitle.test.tsx

```diff
@@ -30,3 +31,15 @@ describe('<WelcomeTitle>', () => {
     expect(getByText(/Get some rest, Guest/)).toBeInTheDocument();
   });
 });
+
+test('should greet user with a single language', async () => {
```

Comment:
Thanks for increasing the code coverage 🙌🏻

Reply:
Anything to do here?

Reply:
No, just sharing appreciation, thank you again 🙂